lib/package/base/server-request/api/zuri_api.dart

```diff
@@ -390,7 +392,6 @@ class ZuriApi implements Api {
   }
 
   // THIS SERVICE IS FOR JOINED ROOMS FOR ACTIVE DMs
-  @override
```

Comment:
With this too

Reply:
Same as above

Reply:
Same with above, nothing for it to override